staging/src/k8s.io/apiserver/pkg/endpoints/filters/audit.go

```diff
@@ -48,37 +48,17 @@ func WithAudit(handler http.Handler, requestContextMapper request.RequestContext
 			return
 		}
 
-		attribs, err := GetAuthorizerAttributes(ctx)
+		ev, err := createAuditEvent(ctx, policy, requestContextMapper, req, w)
 		if err != nil {
```

Comment:
shouldn't this still lead to http 500?

Reply:
Ic, createAuditEvent writes the error. I would prefer to see it here.

Reply:
or at least a good comment here.

Reply:
I'll move the error handling up, it'll be more explicit.